scripts/install_third_party_libs_test.py

```diff
@@ -48,6 +48,7 @@ def setUp(self):
         self.check_function_calls = {
             'check_call_is_called': False,
         }
+        self.call_counter = 0
```

Comment:
I don't think you should have this be a general variable when you use it in only one test.

Use CallCounter in test_utils.py (and possibly self.swap to swap a function with its CallCounter'd-equivalent) instead, in the place where you need to track this.

Reply:
Defined and used it as a local variable now.